modules/serverless-admin-broker-default.adoc

```diff
@@ -0,0 +1,110 @@
+
+Currently, the available broker class options for {ServerlessProductName} are:
+
+* The `MTChannelBasedBroker` class, which uses the default Knative multi-tenant, channel-based broker.
++
+The `config-br-default-channel` config map defines the channel implementation for channel based brokers. This config map is created by default on the cluster when Knative Eventing is installed. This config map must be specified in the `KnativeEventing` CR if you want to set the multi-tenant, channel-based broker as the default broker for a namespace or cluster.
+
+* The `Kafka` broker class, which uses the Knative Kafka broker (currently Technology Preview).
++
+The `kafka-broker-config` config map defines the configuration for Kafka brokers, and is created on the cluster when Knative Kafka is installed. This config map must be specified in the `KnativeEventing` CR if you want to set the Kafka broker as the default broker for a namespace or cluster.
+
+If no broker class is specified, the cluster uses the `MTChannelBasedBroker` class by default.
+
+.Prerequisites
+
+ifdef::openshift-enterprise[]
+* You have cluster administrator permissions on {product-title}.
+endif::[]
+
+ifdef::openshift-dedicated[]
+* You have cluster or dedicated administrator permissions on {product-title}.
+endif::[]
+
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+
+* Optional: If you want to use the Kafka broker, you must install Knative Kafka on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` CR to add the `defaultBrokerClass` spec and `config-br-defaults` config map details:
++
+.Example `KnativeEventing` CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  defaultBrokerClass: <default-broker-class> <1>
+  config:
+    config-br-defaults: <2>
+      default-br-config: |
+        clusterDefault: <3>
+          apiVersion: v1
+          kind: ConfigMap
+          name: config-br-default-channel
+        namespaceDefaults: <4>
+          namespace1:
+            apiVersion: v1
+            kind: ConfigMap
+            name: kafka-broker-config
+            namespace: knative-eventing
+...
+----
+<1> The default broker class for the cluster. This can be either `MTChannelBasedBroker` or `Kafka`. If no broker class is specified, the cluster uses the `MTChannelBasedBroker` class by default.
+<2> Configuration for the `config-br-defaults` config map.
+<3> The cluster-wide default broker class configuration. In this example, the default cluster-wide broker class is `MTChannelBasedBroker` because the `config-br-default-channel` config map is specified.
+<4> The namespace-scoped default broker class configuration. In this example, the default broker implementation for the `knative-eventing` namespace is `Kafka`, because the `kafka-broker-config` config map is specified.
```

Comment:
The wordings in <3> and <4> together seems to imply, that choosing the name of the config map influences the default broker implementation. Is that the intent? ( I don't believe it works that way )

Reply:
@matzew can you clarify?